src/system/PlatformEventSupport.h

```diff
@@ -31,6 +31,10 @@ class PlatformEventing
 public:
     static CHIP_ERROR ScheduleLambdaBridge(System::Layer & aLayer, LambdaBridge && bridge);
     static CHIP_ERROR StartTimer(System::Layer & aLayer, System::Clock::Timeout aTimeout);
+#if CHIP_SYSTEM_CONFIG_USE_LWIP || CHIP_SYSTEM_CONFIG_USE_OPEN_THREAD_ENDPOINT
+    static void LockMatterStack(System::Layer & aLayer);
```

Comment:
what is the use of aLayer here? Can we guarantee that chip stack locking actually exists/is usable before making these available?

Reply:
Removed the aLayer.

> Can we guarantee that chip stack locking actually exists/is usable before making these available?

Yes, added `#if !CHIP_SYSTEM_CONFIG_NO_LOCKING` for it.